docs/index.md

```diff
@@ -51,3 +46,17 @@ Additional documentation on the concepts that support the operation of the main
 * [ipc](details/ipc.md): How all the components of Santa communicate.
   duction/syncing-overview.
 * [logs](details/logs.md): What and where Santa logs.
+
+### Binaries
+
+There are five main components that make up Santa, see the following pages to understand their functionality:
+
+* [santad](details/santad.md): A userland root daemon that makes decisions.
+* [santactl](details/santactl.md): A userland anonymous daemon that communicates with a sync server for configurations and policies. santactl can also be used by a user to manually configure Santa when using the local configuration.
```

Comment:
This description is a little out-dated. Perhaps:

"A command-line utility for inspecting the state of and managing local configuration of Santa"